### ISSUE TITLE
Pre-generated wrapper

### ISSUE THREAD

**Issue by arcvsduo** [StefanSalewski/gintro#3]
Hello,
is there a chance to add pre-generated wrappers? I have hard time with gen.exe on Windows, it fails to find typelibs and I'm not sure where to put them.

**Comment by StefanSalewski** [comment]
Thanks for testing on Windows.

GTK3 on windows is a sad story -- I have no windows on my computer, and I know no one who is using GTK3 applications on windows at all. I am not even sure if there exists GTK3 applications for windows at all -- well I assume that there may exist a few, but a short google search recently did not gave me real results.

When you are SERIOUSLY interested in GTK3 on windows, then we will try  hard to provide you with a solution. But it is not easy. GObject-Introspection generates bindings specific for the box where it is executed, so for 32 and 64 bit is different for sure, and I strongly assume that generated files depend on installed GTK version and maybe on windows version.

Maybe you can start with the low level bindings? They have been available at github.com/ngtk3 for a few years now, and are now available as a nimble package at https://github.com/StefanSalewski/oldgtk3 . That bindings are pregenerated with c2nim and they should work for 64 and 32 bit at the same time, and they should work for windows and mac. But again, no one has tested for mac and windows.

And for the case that you are still really interested and need help: Let us know which windows version you are using, and if you have been able to run any GTK3 application at all on your computer. And if you have used GTK3 bindings of any other programming language on your box, for example for Python, Rust, D...

**Comment by arcvsduo** [comment]
Hello and thank you for the response. I've also always thought that GTK is Linux only thing but considering lack of GUI frameworks for Nim I've decided to try oldgtk3 wrapper and to my surprise it works very well in Windows 7/8.1, at least small subset of controls that I've tried so far (ver. GTK 3.14/GTK 3.22). 
UI feels very fast, it has complete set of controls and can be easily styled with themes which results in beautiful GUI apps in Nim. 

I've thought that problems with generating wrapper are related to my lack of knowledge but if there are such technical implications I will just use oldgtk3 then, thanks! :)
